ts/Core/Templating.ts

```diff
@@ -539,6 +550,14 @@ namespace Templating {
     export interface FormatterCallback<T> {
         (this: T): string;
     }
+    export interface TemplatingOwnerOptions {
```

Comment:
`OwnerOptions` would be sufficient since we're already in the Templating namespace.
```suggestion
    export interface OwnerOptions {
```